src/iqm/qiskit_iqm/fake_backends/fake_apollo.py

```diff
@@ -215,4 +215,4 @@ def IQMFakeApollo() -> IQMFakeBackend:
         name="sample-chip",
     )
 
-    return IQMFakeBackend(architecture, error_profile, name="IQMFakeApolloBackend")
+    return IQMFakeBackend(architecture, error_profile, name="IQMFakeApolloBackend")
```

Comment:
The last (empty) line was accidentally removed, it needs to be added back for the formatting checks to pass.
```suggestion
    return IQMFakeBackend(architecture, error_profile, name="IQMFakeApolloBackend")

```